src/pages/login/index.tsx

```diff
@@ -1,5 +1,16 @@
+import LoginForm from "./component/form";
+
 const Login = () => {
-  return <div className="bg-slate-800 text-white h-[100vh]">Login</div>;
+  return (
+    <div className="h-[100vh] flex items-center justify-center ">
```

Comment:
Recomendo usar as classes do tailwind.
```suggestion
    <div className="h-dvh flex items-center justify-center ">
```